src/redis_cmd.cc

```diff
@@ -4274,10 +4274,12 @@ class CommandFetchFile : public Commander {
             Util::SockSendFile(repl_fd, fd, file_size).IsOK()) {
           LOG(INFO) << "[replication] Succeed sending file " << file << " to "
                     << ip;
-        } else {
-          LOG(WARNING) << "[replication] Fail to send file " << file << " to "
-                       << ip << ", error: " << strerror(errno);
+          close(fd);
+          break;
```

Comment:
We should break the loop when sending the file with an error? or next files will not be sent here.

Reply:
yes

Reply:
So we should NOT break the loop since this condition means sending file was ok.

Reply:
my bad 🤦